### PR TITLE
docs: rewrite readme

### PR DESCRIPTION
**Pull request by ivanagas**
## Problem

A lot of people read our readme, but it doesn't do as much as it could. A better readme would help us stand out more.

https://github.com/PostHog/posthog.com/issues/4708

## Changes

Rewrote readme better accomplish the goal of getting people to use PostHog and update

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
n/a

## Notes
Will need updated graphics from @lottiecoxon (header graphic and maybe a graphic for contributors?)
Would appreciate feedback on:
- Key selling points in intro, what's missing? what could be said better?
- Moving cloud to first option (we might want to still promote hobby deploy first)
- Features, what's missing? What could be said better?
- What do we want to say in the philosophy section? I think we have lots of interesting things to say, but not sure I captured exactly what it is we want to say.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,90 +12,110 @@
...
-* Automatically track every event on your website or app
-* Understand your users and how to improve your product
-* Deploy on your own infrastructure to keep control of your data.
+<p align="center">
+  <a href="https://posthog.com/docs">Docs</a> - <a href="https://posthog.com/using-posthog">Using PostHog</a> - <a href="https://posthog.com/slack">Support community</a> - <a href="https://posthog.com/roadmap">Roadmap</a> - <a href="https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&template=bug_report.md">Bug report</a>
+</p>
+
+## PostHog is an open-source suite of product and data tools, built for engineers
+
+- Automatic and/or customized event capture for your app and website
```
Comment (joethreepwood) [inline README.md:21]: May be worth splitting out into two points?

```suggestion
- Autocapture events, so you spend less time on instrumentation
- Track events manually, so you get the detail you need
```

Alternatively, I like the fact that all other points in this list start with a verb, so making consistent with that:

```suggestion
- Specify events manually, or use autocapture to get started quickly
```

File: README.md [hunk 2]
```diff
@@ -12,90 +12,110 @@
...
+- Use apps to connect to external services and manage data flows
 
+## Table of Contents
 
+- [Get started for free](#get-started-for-free)
+- [Features](#features)
+- [Docs and support](#docs-and-support)
+- [Contributing](#contributing)
+- [Philosophy](#philosophy)
+- [Open-source vs paid](#open-source-vs-paid)
```
Comment (joethreepwood) [inline README.md:34]: May be a stretch goal, but I think could be worth adding a section that compares PostHog to Amplitude, Heap, Logrocket, etc. Could just be a summary, then link to blog content.

File: README.md [hunk 3]
```diff
@@ -12,90 +12,110 @@
...
 
 ## Get started for free
 
-### Option 1: Hobby instance one-line-deploy
-For <100K events ingested monthly on Linux with Docker (recommended 4GB memory):
+### PostHog Cloud
```
Comment (joethreepwood) [inline README.md:38]: Yes, love the reordering and extra detail here.

File: README.md [hunk 4]
```diff
@@ -12,90 +12,110 @@
...
 
-## Event autocapture
-You don't have to spend weeks instrumenting every event on your front-end, point and click at elements from your browser and turn them into events which you and your team can analyze
+- **Event-based analytics:** capture your product's usage [automatically](https://posthog.com/docs/integrate/client/js#autocapture), or [customize](https://posthog.com/docs/integrate) it to your needs
+- **User and group tracking:** understand the [people](https://posthog.com/manual/persons) and [groups](https://posthog.com/manual/group-analytics) behind the events and track properties about them
+- **Data visualizations:** create and share [graphs](https://posthog.com/docs/features/trends), [funnels](https://posthog.com/docs/features/funnels), [paths](https://posthog.com/docs/features/paths), [retention](https://posthog.com/docs/features/retention), and [dashboards](https://posthog.com/docs/features/dashboards)
+- **Session recording:** [watch videos](https://posthog.com/docs/features/session-recording) of your users' behavior, with fine-grained filters and privacy controls
+- **A/B testing:** run changes as experiments and get automatic significance calculations
```
Comment (joethreepwood) [inline README.md:67]: May be worth mentioning we can do multivariant tests too

File: README.md [hunk 5]
```diff
@@ -12,90 +12,110 @@
...
-- **And much much more...** for a [full list of PostHog features](https://posthog.com/features).
+We bring together all the tools and data in one place to help you build better products
 
+### Product analytics and optimization
 
-## Event autocapture
-You don't have to spend weeks instrumenting every event on your front-end, point and click at elements from your browser and turn them into events which you and your team can analyze
+- **Event-based analytics:** capture your product's usage [automatically](https://posthog.com/docs/integrate/client/js#autocapture), or [customize](https://posthog.com/docs/integrate) it to your needs
+- **User and group tracking:** understand the [people](https://posthog.com/manual/persons) and [groups](https://posthog.com/manual/group-analytics) behind the events and track properties about them
+- **Data visualizations:** create and share [graphs](https://posthog.com/docs/features/trends), [funnels](https://posthog.com/docs/features/funnels), [paths](https://posthog.com/docs/features/paths), [retention](https://posthog.com/docs/features/retention), and [dashboards](https://posthog.com/docs/features/dashboards)
```
Comment (joethreepwood) [inline README.md:65]: Correlation analysis may be worth adding as an additional point

File: README.md [hunk 6]
```diff
@@ -12,90 +12,110 @@
...
+- **Plays nicely with data warehouses:** import events or user data from your warehouse by writing a simple transformation plugin, and export data with pre-built apps - such as [BigQuery](https://posthog.com/apps/bigquery-export), [Redshift](https://posthog.com/apps/redshift-export), [Snowflake](https://posthog.com/apps/snowflake-export), and [S3](https://posthog.com/apps/s3-expo)
 
-See [PostHog Docs](https://posthog.com/docs/) for in-depth walk-throughs on functionality.
+[Read a full list of PostHog features](https://posthog.com/features).
```
Comment (joethreepwood) [inline README.md:79]:
```suggestion
[Read a full list of PostHog features](https://posthog.com/product).
```
avoid the redirect if you can

File: README.md [hunk 7]
```diff
@@ -12,90 +12,110 @@
...
 
-Join our [Slack community](https://posthog.com/slack) if you need help, want to chat, or are thinking of a new feature. We're here to help - and to make PostHog even better.
+## Docs and support
 
-## Philosophy
+Read how to [deploy](https://posthog.com/docs/self-host), [integrate](https://posthog.com/docs/integrate), and [extend](https://posthog.com/docs/apps) PostHog in our [documentation](https://posthog.com/docs).
 
-We help you understand user behavior and build better products without losing control of your data. 
+Check out our [tutorials](https://posthog.com/docs/apps) for step-by-step guides, how-to's, and best practices.
```
Comment (joethreepwood) [inline README.md:85]: May need some art assist from Lottie, but I think worth adding the tutorial cover images in as clickable items which break up the text and point to especially good/relevant tutorials. Defer to you.

File: README.md [hunk 8]
```diff
@@ -12,90 +12,110 @@
...
+- Give us feedback in our [Slack community](https://posthog.com/slack)
+- Vote on features in our [roadmap](https://posthog.com/roadmap)
+- Open a PR (see our instructions on [developing PostHog locally](https://posthog.com/handbook/engineering/developing-locally))
+- Submit a [feature request](https://github.com/PostHog/posthog/issues/new?assignees=&labels=enhancement%2C+feature&template=feature_request.md) or [bug report](https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&template=bug_report.md)
 
-We play nicely with data warehouses and other services - you can _import_ event or user data by writing a plugin to create transformations, or you can _export_ data by using our existing data exports to BigQuery, Redshift, Snowflake, etc. All without losing control of your data.
+## Philosophy
 
-## Developing locally & Contributing
+Our mission is to increase the number of successful products in the world. To do that, we continue to build and improve product and data tools that help you understand user behavior without losing control of your data.
```
Comment (joethreepwood) [inline README.md:102]:
```suggestion
Our mission is to increase the number of successful products in the world. To do that, we build and improve product and data tools that help you understand user behavior without losing control of your data.
```

File: README.md [hunk 9]
```diff
@@ -12,90 +12,110 @@
...
+PostHog gives you every tool you need to understand user behavior, develop and test improvements, and release changes to make your product more successful.
 
-Not sure where to start? [Book a free, no-pressure pairing session](mailto:tim@posthog.com?subject=Pairing%20session&body=I'd%20like%20to%20do%20a%20pairing%20session!) with one of the team.
+PostHog operates in public as much as possible. We detail how we work and our learning on building and running a fast-growing, product-focused startup in our [handbook](https://posthog.com/handbook/getting-started/start-here).
 
-## We're hiring!
+## Open-source vs. paid
 
-Come help us make PostHog even better. We're growing fast, [and would love for you to join us](https://posthog.com/careers).
+This repo is entirely [MIT licensed](https://www.notion.so/LICENSE), except for the `ee` directory (if applicable). Need *absolutely 💯% FOSS*? Check out our [posthog-foss](https://github.com/PostHog/posthog-foss) repository, which is purged of all proprietary code and features.
```
Comment (joethreepwood) [inline README.md:112]: This notion link doesn't work and I'm not sure why we link to Notion vs our own license?

**Review by joethreepwood — APPROVED** [review]

File: README.md (hunk 9 above)
Comment (ivanagas) [reply README.md:112]: It's what it was before, I'll rework and link to our license.

**Pull request merged**